core/app/models/spree/address.rb

```diff
@@ -9,7 +9,7 @@ class Address < Spree::Base
     has_many :cartons, inverse_of: :address
     has_many :credit_cards, inverse_of: :address
 
-    validates :firstname, :lastname, :address1, :city, :country, presence: true
+    validates :firstname, :lastname, :address1, :city, :country_id, presence: true
```

Comment:
I am fine with both `validates :country_id` and `validates :country`, just not `validates_associated`